tests/test_interpreter.py

```diff
@@ -72,6 +76,8 @@ def test_windows_python_no_version_with_venv(monkeypatch):
 
 def test_windows_python_no_version_no_venv_with_py(monkeypatch):
     def which(name):
+        if name == "py":
+            return "py"
         return "py"
```

Comment:
This looks like an unnecessary condition?

```suggestion
        return "py"
```